### DISCUSSION TITLE
How to add resize images and add multiple images side by side.

### DISCUSSION THREAD

**Discussion by M-I-Dx** [datapane/datapane#193] [Help]
Hi, 
I am working on a report which will have multiple images and I want them to appear in a per-defined format. For example out of 5 images, 2 appear adjacent to each other while the remain three are also side by side but below the first two images.
Any suggestions.

**Comment by Buroni** [comment]
hey @M-I-Dx, have you tried using `dp.Group`?

```python
dp.Report(
    dp.Group(
        dp.Media(img1),
        dp.Media(img2),
        columns=2,
    ),
    dp.Group(
        dp.Media(img3),
        dp.Media(img4),
        dp.Media(img5),
        columns=3,
    ),
)
```
